packages/schemas/src/util/merge.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Statsify
+ *
+ * This source code is licensed under the GNU GPL v3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * https://github.com/Statsify/statsify/blob/main/LICENSE
+ */
+
+import { RATIOS, RATIO_STATS } from "../ratios";
+import { isObject } from "class-validator";
+import { ratio, sub } from "@statsify/math";
+
+export function merge<T>(oldOne: T, newOne: T): T {
```

Comment:
Rename this to be more specific about merging two historical documents, because `merge` is not very specific